jackson-modules/jackson-annotations/src/test/java/com/baeldung/jackson/deductionBasedPolymorphism/JacksonDeductionBasedPolymorphismBaseUnitTest.java

```diff
@@ -0,0 +1,9 @@
+package com.baeldung.jackson.deductionBasedPolymorphism;
+
+public abstract class JacksonDeductionBasedPolymorphismBaseUnitTest {
+
+  public String a2q(String input) {
+    return input.replaceAll("'", "\"");
+  }
```

Comment:
This utility class should be renamed since it's not a unit test. Moreover, it needs a more meaningful name for the class and the method.